src/langs/json/en.json

```diff
@@ -776,24 +694,7 @@
       }
     },
     "add": "Add",
-    "addCollaborators": "Add Collaborators",
-    "admin": "Admin",
-    "view": "View",
-    "edit": "Edit",
-    "help": {
-      "admin": "Collaborators can edit this project and its documents",
-      "edit": "Collaborators can edit documents in this project",
```

Comment:
I see similar strings in the new version- 
![ViewerEditorAdmin](https://github.com/user-attachments/assets/96e7668d-d5ec-4f6f-86ff-feb24060ab24), but I don't see these strings added?

Reply:
I had a couple versions of these in different places as I was working through the design, so there were duplicates.